Cargo.toml

```
@@ -26,7 +26,7 @@ debug = true
 debug = "line-tables-only"
 
 [workspace.dependencies]
-delta_kernel = { version = "0.1.1" }
+delta_kernel = { version = "0.2.0" }
 # delta_kernel = { path = "../delta-kernel-rs/kernel" }
```

I was recently made aware, that rust allows for multiple specs, it's jsut for my local development, but may be worth considering. :)

```suggestion
delta_kernel = { path = "../delta-kernel-rs/kernel", version = "0.2.0" }
```